newsrevenuehub/rev-engine
# DEV_1837_top_level_links_inherit_donation_page_font

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

#### What's this PR do?
This PR ensures that dashboard "primary navigation" fonts  are always fixed.

#### Why are we doing this? How does it help us?
The dashboard "primary navigation" fonts should always be the same.

#### How should this be manually tested?
Try changing donation page style fonts. The primary navigation fonts should not change as a result.

#### Have automated unit tests been added?
no

#### How should this change be communicated to end users?
n/a

#### Are there any smells or added technical debt to note?
no

#### Has this been documented? If so, where?
no

#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-1837

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

#### Are there next steps? If so, what? Have tickets been opened for them?
